docs/CHILI-GraFx/concepts/sandbox/index.md

```diff
@@ -6,6 +6,10 @@ A sandbox environment is like a test version of the CHILI GraFx platform or appl
 
 This means that any changes made in the sandbox will not affect the production data. The sandbox allows you to test new features to make sure they work properly before they are used in the live version.
 
+Sandboxes will have a watermark in the output and will not count as a [render](/CHILI-GraFx/concepts/renders/).
```

Comment:
I would not say it in the future tense. I would re-phrase it to:
"Output generated in a sandbox environment has a watermark and is not counted as a render."